docs/source/gmaps.rst

```diff
@@ -221,9 +221,11 @@ Markers are currently limited to the Google maps style drop icon. If you need to
 
     gmaps.configure(api_key="AI...")
 
-    starbucks_locations = gmaps.datasets.load_dataset("starbucks_uk")
+    df = gmaps.datasets.load_dataset_as_df("starbucks_kfc_uk")
+    starbucks_df = df.query("chain_name == 'starbucks'").drop("chain_name", axis=1)
```

Comment:
I consider the `.query` notation a little bit more readable but that is not sufficient to beat consistency with the style in `starbucks_df[['latitude', 'longitude']]`. So, I reverted to the previous syntax - also dropped `.drop()`.